src/expr/src/scalar/func.rs

```diff
@@ -2389,8 +2413,18 @@ impl BinaryFunc {
             BinaryFunc::AddInt64 => eager!(add_int64),
             BinaryFunc::AddFloat32 => eager!(add_float32),
             BinaryFunc::AddFloat64 => eager!(add_float64),
-            BinaryFunc::AddTimestampInterval => eager!(add_timestamp_interval),
-            BinaryFunc::AddTimestampTzInterval => eager!(add_timestamptz_interval),
+            BinaryFunc::AddTimestampInterval => {
+                eager!(|a: Datum, b: Datum| add_timestamplike_interval(
+                    a.unwrap_timestamp(),
+                    b.unwrap_interval(),
+                ))
+            }
+            BinaryFunc::AddTimestampTzInterval => {
```

Comment:
The variant for timestamptz is not documented.

Reply:
Because we support the same functions for timestamp and timestamptz, we infrequently document both variants.